llvm/lib/Support/Yk.cpp

```diff
@@ -18,3 +18,9 @@ static cl::opt<bool, true> YkExtendedLLVMBBAddrMapSectionParser(
     "yk-extended-llvmbbaddrmap-section",
     cl::desc("Use the extended Yk `.llvmbbaddrmap` section format"),
     cl::NotHidden, cl::location(YkExtendedLLVMBBAddrMapSection));
+
+bool YkStackMapOffsetFix;
+static cl::opt<bool, true> YkStackMapOffsetFixParser(
+    "yk-stackmap-offset-fix",
+    cl::desc("Apply a fix to stackmaps that corrects the reported instruction offset in the presence calls."),
```

Comment:
presence *of* calls?

Reply:
Fixed in [e81d2f8](https://github.com/ykjit/ykllvm/pull/52/commits/e81d2f87d0e60a96fd2274aecbceff64fc4a1ee3)